src/UI/DialogBox.java

```diff
@@ -6,16 +6,19 @@
 import java.awt.event.ActionListener;
 
 public class DialogBox implements ActionListener {
-    JFrame jframe;
+    public JFrame jframe;
```

Comment:
Here's another public variable -- all instance variables should be private with a setter and/or a getter if needed outside the class.